packages/readable-js-assets-webpack-plugin/test/fixtures/webpack.config.js

```diff
@@ -17,9 +17,7 @@ module.exports = {
 		concatenateModules: true,
 		minimizer: [
 			new TerserPlugin( {
-				cache: true,
```

Comment:
@jsnajdr, is it correct? I had to remove both flags because they were removed in webpack 5. The output generated in the tests needs to be verified, too.